library/src/main/java/com/airbnb/android/airmapview/MapboxWebViewMapFragment.java

```diff
@@ -22,4 +22,8 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle sa
 
     return view;
   }
+
+  @Override public void setMapType(MapType type) {
+    // TODO: Not implemented
```

Comment:
@bleege I can't seem to find a way to make it work. `map.id = 'mapbox.sattelite` has no effect.